Cannot install webext-redux next to react-redux

I really just ran `npm i react-redux` and then `npm i webext-redux`.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: react-redux@9.1.0
npm ERR! Found: redux@4.2.1
npm ERR! node_modules/redux
npm ERR!   peer redux@">= 3 <= 4" from webext-redux@2.1.9
npm ERR!   node_modules/webext-redux
npm ERR!     webext-redux@"*" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional redux@"^5.0.0" from react-redux@9.1.0
npm ERR! node_modules/react-redux
npm ERR!   react-redux@"^9.1.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: redux@5.0.1
npm ERR! node_modules/redux
npm ERR!   peerOptional redux@"^5.0.0" from react-redux@9.1.0
npm ERR!   node_modules/react-redux
npm ERR!     react-redux@"^9.1.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! /home/panda/.npm/_logs/2024-03-04T19_42_46_254Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /home/panda/.npm/_logs/2024-03-04T19_42_46_254Z-debug-0.log
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "babel-eslint": "^7.2.0",
     "eslint": "^4.18.2",
     "mocha": "^5.2.0",
-    "redux": "^4.0.4",
+    "redux": "5.0.1",
     "rollup": "^1.22.0",
     "rollup-plugin-babel": "^4.3.3",
     "rollup-plugin-commonjs": "^10.1.0",
@@ -49,6 +49,6 @@
     "sinon": "^6.0.0"
   },
   "peerDependencies": {
-    "redux": ">= 3 <= 4"
+    "redux": ">= 3 <= 5"
   }
 }

